src/command_modules/azure-cli-eventhubs/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.4.0
++++++
+* Added property 'enable-kafka' in namespace to support Enabling Kafka
```

Comment:
This should be 0.3.1 and the format should be:
`eventhub namespace create/update`: Added `--enable-kafka` flag to support Kafka.

Reply:
Resolved